### PR TITLE
Just Made `ticket` Name Shorter!

### PR DESCRIPTION
**Pull request by MrYogi360**
In the ticket function, the channel name was too long!
Just made it short

### CODE REVIEW

**Comment by A-K-O-R-A** [comment]
Please contribute to the [dev](https://github.com/Rahuletto/simply-djs/tree/dev) branch instead of the [main](https://github.com/Rahuletto/simply-djs/tree/dev) branch

File: src/clickBtn.js [hunk 1]
```diff
@@ -53,7 +53,12 @@ async function clickBtn(button, options = []) {
       let { MessageButton, MessageActionRow } = require("discord.js");
 
       if (button.customId === "create_ticket") {
-        let ticketname = `ticket_${button.user.id}`;
+        
+        let split = '';
+        let usr = button.user.id.split(split);
+        for (var i = 0; i < usr.length; i++) usr[i] = usr[i].trim();
+
+        let ticketname = `ticket-${usr[0]}${usr[1]}${usr[2]}${usr[3]}`
```
Comment (A-K-O-R-A) [inline src/clickBtn.js:61]: You could replace these 6 lines with this one, which improves readability a lot.
```js
let ticketname = button.user.id.slice(0, 4)
```

**Comment by Assassin-1234** [comment]
I dont think this is the best way, you should start with a 2 - 4 digit number and increment it upon created tickets, would be more efficient imo

**Comment by MrYogi360** [comment]
> I dont think this is the best way, you should start with a 2 - 4 digit number and increment it upon created tickets, would be more efficient imo

wdym? 😅

**Comment by MrYogi360** [comment]
> Please contribute to the [dev](https://github.com/Rahuletto/simply-djs/tree/dev) branch instead of the [main](https://github.com/Rahuletto/simply-djs/tree/dev) branch

done! 👍

**Pull request closed without merging**